forge/Installation and Upgrading/installubuntuapachemysql.md

```diff
@@ -129,8 +106,10 @@ You can save the file now. Some explanations about this Virtual Host configurati
 
 Then you need to activate the site and reload the apache2 configuration
 
-    sudo a2ensite taoplatform
-    sudo service apache2 reload
+```shell
+sudo a2ensite taoplatform
```

Comment:
in the old documentation, it's written : 
_sudo a2ensite taoplatform
sudo service apache2 reload_
I think it worked when I used that command.